### ISSUE TITLE
element freezes session

### ISSUE THREAD

**Issue by Stormfinger** [vector-im/element-web#24407]

**Comment by robintown** [comment]
Hi @Stormfinger, to be able to determine the cause of this issue we will need more information. The next time you manage to get the app started, could you please submit logs from the 'Feedback' menu?

**Comment by Stormfinger** [comment]
i checked info and log would send some information that is not realy classified but i wouldnt like to share... also can note that i dont have strong GPU in sistem as i saw in somewhere there is kinda no gpu option ..ill try that first

**Comment by MadLittleMods** [comment]
Maybe related to https://github.com/vector-im/element-web/issues/5315

**Comment by Stormfinger** [comment]
yes ..thank you indeed it could be notification issue at least most of the time.
